### ISSUE TITLE
[Bug]: Topoislands as mentioned in the README_ISLANDS file does not appear to be a valid command

### ISSUE THREAD

**Issue by 2ndRecon** [DarkPacks/SkyFactory-One#182]
### General Info

- [ ] I can reproduce this issue consistently in single-player
- [X] I can reproduce this issue consistently in multi-player
- [X] I have searched for this issue previously and it was either (1) not previously reported, or (2) previously fixed and I am having the same problem.
- [ ] I am crashing and can provide my crash report(s)
- [ ] I am using the latest version of the modpack

### Modpack version

1.0.5

### Java version

Java 8

### Modpack Launcher Used

Curse

### Memory Allocated

8 GB

### Minecraft World Environment

LAN

### Misc Client Info

- [ ] Optifine Installed
- [ ] Shaders Installed

### Server Java Version

Java 8

### Server Operating System

Ubuntu 22

### Misc Server Info

- [ ] Sponge or Non-Vanilla Forge Server

### Issue Description

README_ISLANDS States to run topoislands command to assign player to island, or from OP, run this command to assign / create island for player.

This doesn't appear to be a valid command

**Comment by sam-kirby** [comment]
You need to add the mod to the server yourself as the pack developers decided not to include it: https://www.curseforge.com/minecraft/mc-mods/topography-islands/files/3640071

**Comment by artdude543** [comment]
Yeah, sadly the mod needs to be downloaded manually. It used to be shipped in the server files but during some transitions to new builders (due to CF API changes), it got lost in the pipe-works.

I'll make sure that any future updates will include the mod.
